### ISSUE TITLE
Change checkboxes to icons upper left corner

### ISSUE THREAD

**Issue by Rapdrei** [05dirnbe/nefi#2]
We need to change the checkboxes for the immediate results in the upper left corner to icons to make them more visual appealing.

**Comment by LumPenPacK** [comment]
Icons may improve the visual appearance a bit but I'm still not sure if this would be the best solution. Probably we should get rid of the whole left upper clutter or move it to another position. The problem with icons is they can't give you a clear information about the current state (on/off). 
There might be an event better solution for this. Maybe we need to think a little bit more about this.

**Comment by Rapdrei** [comment]
Let's put them into the menubar under a menuitem view perhaps. The user will click these options very infrequent and keep his settings for a while. This way we dont use the icons and have no undpleasent looking content in the upper left.

**Comment by Rapdrei** [comment]
The only thing we should keep is the trash bin. I dont know where we should put it. It might look a bit lonly in the upper left corner.

**Comment by tastyminerals** [comment]
Here are a few icons.
**Auto clear** ![auto_clear](https://cloud.githubusercontent.com/assets/7676160/14179009/9ac6685e-f75d-11e5-9e7a-129fd24f09ff.png) 
**Auto clear 2** ![auto_clear2](https://cloud.githubusercontent.com/assets/7676160/14179010/9aeb5556-f75d-11e5-924b-66be12b7672d.png)
**Auto scroll** ![auto_scroll](https://cloud.githubusercontent.com/assets/7676160/14179013/9aef9c10-f75d-11e5-8207-77ef0f7da3c3.png)
**Auto scroll 2** ![auto_scroll2](https://cloud.githubusercontent.com/assets/7676160/14179012/9aef1b8c-f75d-11e5-88fa-039c206cb5f0.png)
**Results only** ![res_only](https://cloud.githubusercontent.com/assets/7676160/14179014/9af2def2-f75d-11e5-98fe-c112de87c6c9.png)
**Results only 2** ![res_only2](https://cloud.githubusercontent.com/assets/7676160/14179011/9aedec6c-f75d-11e5-833f-b12acb5e5879.png)

**Comment by 05dirnbe** [comment]
I must say that i like the brush, the downwards arrows (autoscroll 2 )and the target symbol. When i said that ideally, the buttons give a visual indication wheter they are active i meant something like this:

[http://creativenerds.co.uk/wp-content/uploads/2011/02/letterpressed.jpg](url)

Keep in mind though, that due to our apparent fixation on dark themes, the color grey is tricky. I am convinced though, that you will resolve this issue quite fine.

**Comment by Rapdrei** [comment]
So our idea is:
we take the checkboxes with the labels and place them in the menubar under the menuitem "view". Additionally we put icons into QPushbuttons and give them two different visual states (just to display that a button is pressed or not) as well as a tooltip. The resulting buttons will be in one line with the trash bin for the immediate results so everything looks clean and nice.

**Comment by 05dirnbe** [comment]
Sounds good. Make it so.

**Comment by Rapdrei** [comment]
So we decided now to go with a more minimalistic approach. We moved the checkboxes in the menu option View and deleted them from the immediate results panel. Afterwards the trash bin symbol moved to the bottom end of the immediate results panel. This way we have a symmetric finish between immediate results panel and midpanel.
![capture111](https://cloud.githubusercontent.com/assets/9344404/14409509/541580ec-ff15-11e5-8f20-2cf94d0c5ea7.PNG)
